Deeploy/Targets/Neureka/TileConstraints/RequantHelpers.py

```
@@ -0,0 +1,94 @@
+def serializeTilingSolution(
+        tilingSolution: NodeMemoryConstraint, absoluteOutputCubes: List[AbsoluteHyperRectangle], targetMemLevel: str,
+        ctxt: NetworkContext,
+        operatorRepresentation: OperatorRepresentation) -> Tuple[VariableReplacementScheme, TilingSchedule]:
+    variableReplacementSchedule, tilingSchedule = super().serializeTilingSolution(tilingSolution, absoluteOutputCubes,
+                                                                                  targetMemLevel, ctxt,
+                                                                                  operatorRepresentation)
+
+    outputCubes = [cube.rectangle for cube in absoluteOutputCubes]
+
+    addrNames = ['mul', 'add']
+    inputRequantBaseOffsets, _ = cls.extractBaseAddr(tilingSolution, targetMemLevel, operatorRepresentation, addrNames)
```

Ruff (0.14.0)

Undefined name `cls`

(F821)